packages/preact/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@prairielearn/preact",
+  "version": "1.0.0",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PrairieLearn/PrairieLearn.git",
+    "directory": "packages/preact"
+  },
+  "exports": {
+    ".": "./dist/index.js",
+    "./client": "./dist/client.js",
+    "./hydrated-component": "./dist/hydrated-component/index.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch --preserveWatchOutput & tscp --watch"
+  },
+  "dependencies": {
+    "@prairielearn/browser-utils": "workspace:^",
+    "@prairielearn/compiled-assets": "workspace:^",
+    "@prairielearn/error": "workspace:^",
+    "@prairielearn/html": "workspace:^",
+    "@prairielearn/preact-cjs": "workspace:^",
+    "clsx": "^2.1.1",
+    "preact": "link:../preact-cjs",
```

Comment:
Will this do something sensible when Yarn is publishing this to npm? I'm not convinced it will.

Reply:
Changed to a normal semver range in 0ce1895.